### ISSUE TITLE
unrecognized argument: --decorate-refs=refs/tags/*

### ISSUE THREAD

**Issue by fredrikaverpil** [mtkennerly/dunamai#11]
We just recently started seeing this error and I have not yet concluded that it's because of the recent dunamai release, but I suspect it can be related so I am just quickly writing this ticket here:

```bash
$ poetry install -v --no-root

...

[RuntimeError]
The command 'git log --simplify-by-decoration --topo-order --decorate=full "--decorate-refs=refs/tags/*" HEAD "--format=%H%d"' returned code 128. Output:
fatal: unrecognized argument: --decorate-refs=refs/tags/*
```

This is with:
* poetry==1.0.10
* poetry-dynamic-versioning==0.9.0

**Comment by mtkennerly** [comment]
Thank you for letting me know. That's definitely from the new version, so I'll release a fix ASAP. (I really need to set up Git 2.7.0 for testing in CI to keep this from happening.)

**Comment by mtkennerly** [comment]
I've just released v1.5.2 to avoid `--decorate-refs`. Unfortunately, I wasn't yet able to add Git 2.7.0 to the testing, since (I think) it requires an older OpenSSL, which is giving me some trouble. If you see any further issues with the new release, please let me know.

**Comment by fredrikaverpil** [comment]
Great, thanks 👍
